src/storage/index/hash_index_builder.cpp

```diff
@@ -172,6 +193,68 @@ void HashIndexBuilder<T>::flush() {
     }
 }
 
+template<typename T>
+void HashIndexBuilder<T>::rehashSlots(slot_id_t primarySlotId) {
```

Comment:
should we try to cover this code path?

Reply:
Looks like the rdf test dataset doesn't trigger the rehash.

Reply:
Can we not introduce changes to hash index now? I feel this should be done later in a more careful way.